src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from 'react';
+
+type Props = {
+  errorMessage: string;
+  onClose: () => void;
+};
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  onClose,
+}) => {
+  useEffect(() => {
+    if (!errorMessage) {
+      return;
+    }
+
+    const timer = setTimeout(() => {
+      onClose();
+    }, 3000);
+
+    return () => clearTimeout(timer);
+  }, [errorMessage]);
```

Comment:
fix linter error
onClose also shoule be in dependencies.
![Screenshot 2025-06-28 at 08 59 18](https://github.com/user-attachments/assets/b9ada6dd-3957-46b7-be7a-4e6a6c987f4d)

make sure onClose function is memoised